Unable to pass multi-pow test item with FPGA accelerator

Error messages with FPGA accelerator:

```
root@lampa:~/dcurl# make BUILD_FPGA_ACCEL=1 check                               
make: Warning: File 'mk/python.mk' has modification time 1534568560 s in the fue
  CC    build/curl.o                                                            
  CC    build/constants.o                                                       
  CC    build/trinary.o                                                         
  CC    build/dcurl.o                                                           
  CC    build/implcontext.o                                                     
  CC    build/pow_c.o                                                           
  CC    build/pow_fpga_accel.o                                                  
  CC    build/test-trinary.o                                                    
  LD    build/test-trinary                                                      
*** Validating build/test-trinary ***                                           
        [ Verified ]                                                            
  CC    build/test-curl.o                                                       
  LD    build/test-curl                                                         
*** Validating build/test-curl ***                                              
        [ Verified ]                                                            
  CC    build/test-dcurl.o                                                      
  LD    build/test-dcurl                                                        
*** Validating build/test-dcurl ***                                             
[  409.187196] Get interrupt                                                    
        [ Verified ]                                                            
  LD    build/libdcurl.so                                                       
*** Validating build/test-multi-pow ***                                         
[  413.727193] Get interrupt                                                    
[  413.745242] Get interrupt                                                    
[  413.992183] Get interrupt                                                    
[  414.190852] Get interrupt                                                    
[  414.291223] Get interrupt                                                    
[  414.386504] Get interrupt                                                    
[  414.517357] Get interrupt                                                    
Unhandled exception in thread started by <function call_dcurl at 0xb5a75300>    
Traceback (most recent call last):                                              
  File "./build/test-multi-pow", line 47, in call_dcurl                         
    trytes = TryteString(ret[:2673])                                            
TypeError: 'NoneType' object is not subscriptable 
```
and then type ctrl+c

```
Traceback (most recent call last):                                            
  File "./build/test-multi-pow", line 80, in <module>                           
    testing()                                                                   
  File "./build/test-multi-pow", line 71, in testing                            
    while join_list[i].locked(): pass                                           
KeyboardInterrupt                                                               
mk/common.mk:27: recipe for target 'build/test-multi-pow.done' failed           
make: *** [build/test-multi-pow.done] Interrupt                                 
make: *** Deleting intermediate file 'build/test-trinary.o'                     
make: *** Deleting intermediate file 'build/test-curl.o'                        
make: *** Deleting intermediate file 'build/test-dcurl.o'  
```

--- a/src/pow_fpga_accel.c
+++ b/src/pow_fpga_accel.c
@@ -54,11 +54,11 @@ static bool PoWFPGAAccel(void *pow_ctx)
     Trits_t *object_trits = trits_from_trytes(object_trytes);
     if (!object_trits)
         return false;
-    
-    lseek(ctx->in_fd,0,0);
-    lseek(ctx->ctrl_fd,0,0);
-    lseek(ctx->out_fd,0,0);
-    
+
+    lseek(ctx->in_fd, 0, 0);
+    lseek(ctx->ctrl_fd, 0, 0);
+    lseek(ctx->out_fd, 0, 0);
+
     if (write(ctx->in_fd, (char *) object_trits->data, transactionTrinarySize) <
         0)
         return false;

--- a/src/pow_fpga_accel.c
+++ b/src/pow_fpga_accel.c
@@ -54,7 +54,11 @@ static bool PoWFPGAAccel(void *pow_ctx)
     Trits_t *object_trits = trits_from_trytes(object_trytes);
     if (!object_trits)
         return false;
-
+    
+    lseek(ctx->in_fd,0,0);
+    lseek(ctx->ctrl_fd,0,0);
+    lseek(ctx->out_fd,0,0);
+    
     if (write(ctx->in_fd, (char *) object_trits->data, transactionTrinarySize) <
         0)
         return false;

